eos-copy-host-mali-driver.service

```diff
@@ -0,0 +1,21 @@
+# Copy the host GL driver for Mali to a place where Flatpak will mount it
+
+[Unit]
+Description=Copy Mali GL driver for Flatpak
+DefaultDependencies=no
+Conflicts=shutdown.target
+Wants=local-fs.target
+After=local-fs.target
+Before=multi-user.target
+
+# Only run on ARM systems with the Mali driver
+ConditionArchitecture=arm
+ConditionPathExists=/usr/lib/arm-linux-gnueabihf/mali400-egl-x11
+
```

Comment:
yeah but as you have it now, it will run fork the shell needlessly every boot, adding to boot delay and the boot IO storm... we really want another condition which looks for whether the link exists. maybe if we create the link in both /var/lib and /var/endless-extra we can use the existence of the former to check whether we should be running?

(and I still think creating the link in the image builder is worthwhile.)